android/quest/src/main/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensions.kt

```diff
@@ -71,7 +74,20 @@ fun List<ActionConfig>.handleClickEvent(
             bundleOf(
               NavigationArg.PROFILE_ID to id,
               NavigationArg.RESOURCE_ID to resourceData?.baseResourceId,
-              NavigationArg.RESOURCE_CONFIG to actionConfig.resourceConfig
+              NavigationArg.RESOURCE_CONFIG to actionConfig.resourceConfig,
+              NavigationArg.PARAMS to
+                actionConfig
+                  .params
+                  .map {
+                    ActionParameter(
+                      key = it.key,
+                      paramType = it.paramType,
+                      dataType = it.dataType,
+                      linkId = it.linkId,
+                      value = it.value.interpolate(resourceData?.computedValuesMap ?: emptyMap())
+                    )
+                  }
+                  .toTypedArray()
```

Comment:
is this duplicating 49-60? Also maybe duplicating 125-139 in ProfileViewModel, https://github.com/opensrp/fhircore/pull/2070/files#diff-8d649a3643b25b19e29bc3d0bdc0583811450a85fd92e128e6a6615f8fae07fbR125